modules/MatchProvider.js

```diff
@@ -5,7 +5,8 @@ import {
 
 class MatchProvider extends React.Component {
   static childContextTypes = {
-    match: matchContextType.isRequired
+    match: matchContextType.isRequired,
```

Comment:
I'll hold off on changing this, but I agree that `match` is ambiguous and a different name might be preferable.